source/whats-new.txt

```diff
@@ -30,6 +31,32 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _csharp-version-2.27:
+
+What's New in 2.27
+------------------
+
+The 2.27 driver release includes the following new features:
+
+- Adds support for the ``$sample`` aggregation operator.
+
+- Implements the ``Equals()`` method for serializers in LINQ3.
+
+- Includes tests confirming that read and write concerns are not
```

Comment:
Not just tests.
"Avoid appending Write/Read Concern in Atlas Search Index Helper commands"

Reply:
Reworded - let me know what you think!